pkg/cgroups/cgroups.go

```diff
@@ -131,17 +56,6 @@ func CheckOrMountCgrpFS(mapRoot string, runsOnKind bool) {
 		// Failed cgroup2 mount is not a fatal error, sockmap will be disabled however
 		if err == nil {
```

Comment:
Can you add a log message for the error condition? :pray:

Reply:
This is fixed in the PR - https://github.com/cilium/cilium/pull/15999.